src/material/_index.scss

```diff
@@ -18,7 +18,7 @@
 @forward './core/typography/typography' show typography-hierarchy;
 @forward './core/typography/typography-utils' show font-shorthand;
 @forward './core/tokens/m2' show m2-tokens-from-theme;
-@forward './core/tokens/m3-system' show system-level-colors,
+@forward './core/tokens/m3-system' show theme, system-level-colors,
```

Comment:
If you sync up to `main`, youll see this is currently exported as `private-experimental-theme` so we can develop code for this without any expectation for users yet. Once we get things fully fleshed out over the next couple weeks, we can change this to be `theme`

Reply:
Good to know, I also updated the README with a note about it